content/ops/troubleshooting-bosh.md

```diff
@@ -5,49 +5,72 @@ menu:
 title: Troubleshooting Bosh
 ---
 
-## Accessing Jump Boxes
+## Accessing jumpboxes
 
-If you're going to be accessing Bosh, you will need to intercept a jumpbox via Concourse.
+If you're going to be accessing Bosh, you will need to intercept a jumpbox via
+Concourse.
 
 ### Pre-flight checklist:
+
 1. Need to have access to Concourse for cloud.gov (e.g., https://ci.cloud.gov)
 1. Need to have `fly` installed locally
```

Comment:
It's in the bottom corner of ci.[fr.].cloud.gov, and yeah, we should point that out explicitly.